src/readme.graph.md

```diff
@@ -165,6 +165,11 @@ directive:
       subject: ^(.*)List(.*)(As.*)$
     set:
       subject: $1$2$3
+  - where:
+      verb: Get|New|Remove|Update
```

Comment:
We can exclude the verb since we want the rename to apply to commands verbs (HTTP methods + OData actions and functions)
```suggestion
```